packages/x-components/src/components/animations/collapse-height.vue

```diff
@@ -29,6 +29,14 @@
     inheritAttrs: false
   })
   export default class CollapseHeight extends Vue {
+    /**
+     * Indicates if the transition must be applied on the initial render of the node.
+     */
+    @Prop({
+      type: Boolean,
```

Comment:
Do we need to specify the type here? I think that it looks redundant as it is already typed in line 39. What do you think? @ajperezbau